packages/flutter/lib/src/services/sensitive_content.dart

```diff
@@ -0,0 +1,81 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// @docImport 'package:flutter/widgets.dart';
+library;
+
+import 'package:flutter/foundation.dart';
+
+import 'system_channels.dart';
+
+/// The level of sensitivity that content in a particular Flutter view
+/// contains.
+/// 
+/// * See [SensitiveContent] for how to set a [ContentSensitivity] level
+///   in order for sensitive content to be obscured when the Flutter screen
+///   is shared. 
+enum ContentSensitivity {
```

Comment:
I think that the "severity", or the behavior that certain values here override other values, is something that we need to make sure is clearly documented to the user. You explain this well in the docs for each enum value below, but I worry that it's less likely users will find that info there. It's probably worth either summarizing this directly in the docs for the SensitiveContent widget, or referring to this or something else in a "See also". Some other places might be worth adding a "See also" too, like SensitiveContent.sensitivityLevel and maybe these top level ContentSensitivity docs itself.